Assignments/A3.py

```diff
@@ -0,0 +1,54 @@
+import sys
+
+from transform import rollingTransform
+from pyspark import StorageLevel
+from pyspark.ml import Pipeline
+from pyspark.sql import SparkSession
+
+
+def main():
+
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+    url = "jdbc:mysql://localhost:3306/baseball"
+
+    gameq = """
+            SELECT game_id, DATE(local_date) AS game_date
+            FROM game
+            """
+
+    batter = """
+            SELECT game_id, batter, Hit AS hit, atBat AS atbat
+            FROM batter_counts WHERE atBat > 0
+            """
+# Enter your username and password to connect to baseball database
+
+    user = "YourUserHere"
```

Comment:
Maybe include an echo in the sh script letting the user know they need to update db creds in the py file. Not strictly necessary for the assignment probably, but just an idea.